e2e-tests/cypress/tests/integration/channels/enterprise/elasticsearch_autocomplete/helpers/index.js

```diff
@@ -1,246 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
There are two tests outside of `elasticsearch_autocomplete/` that depend on this file:

- /channels/search_autocomplete/channels_spec.js
- /channels/search_autocomplete/renaming_spec.js

Could you make sure they use the newly renamed doc (`elasticsearch_autocomplete/helpers.ts`)?

Reply:
Thansk for the review.
I have added some missing exports.